### ISSUE TITLE
Datepicker doesn't exit on mobile when selecting date and time.

### ISSUE THREAD

**Issue by dilizarov** [Hacker0x01/react-datepicker#1222]
### Expected behavior

When I select a date and then a time, the Datepicker should close

### Actual behavior

When I select a date and then a time, the Datepicker stays open

### Steps to reproduce

```
<DatePicker
      selected={props.startDate}
      onChange={props.changeStartDate}
      showTimeSelect
      timeIntervals={30}
      dateFormat="MM/DD/YY, hh:mm a"
      todayButton="Today"
      minDate={moment()}
      readOnly
       />
```

I have the latest version. I'm not sure why this is happening.

**Comment by aimarjs** [comment]
Same here!

**Comment by mrchief** [comment]
I'm seeing this on dekstop as well.

**Comment by wengkhing** [comment]
I experience this problem using mobile Chrome browser and also desktop Chrome with device mode on. Couldn't reproduce this problem at Firefox

**Comment by wengkhing** [comment]
I find that adding 'preventOpenOnFocus' to props will solve this problem
`
<DatePicker
      selected={props.startDate}
      onChange={props.changeStartDate}
      showTimeSelect
      timeIntervals={30}
      dateFormat="MM/DD/YY, hh:mm a"
      todayButton="Today"
      minDate={moment()}
      readOnly
      preventOpenOnFocus
       />
`

**Comment by chandawarlokesh** [comment]
I have resolved this issue and already created PR #1400 

Use of preventOpenOnFocus is not used that much if my requirement is related to the focus event like in the form after every field user generally used the tab to move next.
